example/src/api/events.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response, Router } from 'express';
+import { createEvent, editEvent, EventStatus, getEvent, getManyEvents } from '@jiter/node';
+
+export const events = Router();
+
+// See https://docs.jiter.dev/docs/rest-api/get-many-events
+events.get('/', async (req: Request, res: Response) => {
+  try {
+    const allEvents = await getManyEvents();
+    res.send(allEvents);
+  } catch (error) {
+    console.error(error);
+  }
+});
+
+// See https://docs.jiter.dev/docs/rest-api/get-event-info
+events.get('/:id', async (req: Request, res: Response) => {
+  try {
+    const event = await getEvent({
+      id: req.params.id,
+    });
+    res.send(event);
+  } catch (error) {
+    console.error(error);
+  }
```

Comment:
```suggestion
  } catch (error: any) {
    console.error(error);
    const { message } = error.response.data;
    res.status(error.status).json({ message, error });
  }
```

This will hang forever